src/github.com/getlantern/flashlight/statserver/peer.go

```diff
@@ -10,7 +10,7 @@ import (
 
 var (
 	publishInterval   = 10 * time.Second
-	retryWaitTime     = 100 * time.Millisecond
+	retryWaitTime     = 500 * time.Millisecond
```

Comment:
Maximum wait time between tries was 2^9*100 ms (51s). This value was probably too small. I'm pushing it to 2^9*500ms (4.27m).